save-orchestrator/src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 orchestrator.backendUrl=http://backend:5000
+orchestrator.agentLogs=/home/cnb/agentLogs/
```

Comment:
One last thing - let's rename these directories and call them `executionLogs`, similar to endpoint name. Because the logs won't be from agent, they'll be logs from CLI, provided by agent; and with current folder name it would be confusing.

Reply:
Ok, renamed